spec/rubocop/cop/style/multiline_list_spec.rb

```diff
@@ -0,0 +1,483 @@
+# encoding: utf-8
+
+require 'spec_helper'
+
+describe RuboCop::Cop::Style::MultilineList, :config do
+  subject(:cop) { described_class.new(config) }
+
+  context 'method parameters' do
+    let(:cop_config) do
+      {
+        'EnableMethodParams' => true
+      }
+    end
+
+    context 'params listed on the first line' do
+      let(:source) do
+        ['def foo(bar,',
+         '  baz)',
+         '  do_something',
+         'end']
+      end
+
+      it 'detects the offense' do
+        inspect_source(cop, source)
+
+        expect(cop.offenses.length).to eq(1)
+        expect(cop.offenses.first.line).to eq(1)
+        expect(cop.highlights).to eq(['bar'])
+      end
+
+      it 'autocorrects the offense' do
+        new_source = autocorrect_source(cop, source)
+
+        expect(new_source).to eq(
```

Comment:
It relies on `Style/FirstParameterIndetation`:

http://www.rubydoc.info/gems/rubocop/0.29.1/RuboCop/Cop/Style/FirstParameterIndentation